cavern/src/main/java/org/opencadc/cavern/files/FileAction.java

```diff
@@ -83,22 +83,21 @@
 import org.opencadc.vospace.server.LocalServiceURI;
 import org.opencadc.vospace.server.NodePersistence;
 import org.opencadc.vospace.server.PathResolver;
+import org.opencadc.vospace.server.actions.NodeAction;
 import org.opencadc.vospace.server.auth.VOSpaceAuthorizer;
 
 /**
  *
  * @author majorb
  * @author jeevesh
  */
-public abstract class FileAction extends RestAction {
+public abstract class FileAction extends NodeAction {
```

Comment:
No - A FilesAction is not a NodeAction and should not subclass just because they happen to have some code in common.

Reply:
Agree.